Jenkinsfile.d/package

```diff
@@ -226,19 +234,36 @@ pipeline {
             }
             stage('Publish'){
               steps {
-                dir (WORKING_DIRECTORY){
-                  powershell '''
-                    & .\\msi\\publish\\publish.ps1
-                  '''
-                  archiveArtifacts 'msi\\build\\bin\\Release\\en-US\\*.msi'
-                  archiveArtifacts 'msi\\build\\bin\\Release\\en-US\\*.msi.sha256'
+                sshagent(['pkgserver']) {
+                  dir (WORKING_DIRECTORY){
+                    powershell '''
+                      & .\\msi\\publish\\publish.ps1
+                    '''
+                    archiveArtifacts 'msi\\build\\bin\\Release\\en-US\\*.msi'
+                    archiveArtifacts 'msi\\build\\bin\\Release\\en-US\\*.msi.sha256'
+                  }
                 }
               }
             }
           }
         }
       }
     }
+    // Force mirror synchronization
+    stage('Synchronize mirror'){
+      environment {
+        PKGSERVER = "mirrorbrain@pkg.jenkins.io"
+      }
+      steps{
+        container('jnlp'){
+          sshagent(['pkgserver']) {
+            sh '''
+              ssh $PKGSERVER /srv/releases/sync.sh
```

Comment:
this will fail in the current state afaik due to it using the live server but prob fine..

Reply:
Are you referencing pkg.jenkins.io ? I created a simple script on the azure machine that just say `echo bla`